guide/selenium/src/test/scala/io/udash/web/guide/demos/ext/I18nDemosTest.scala

```diff
@@ -9,19 +9,11 @@ class I18nDemosTest extends SeleniumTest {
   override protected final val url = "/ext/i18n"
 
   "I18n view" should {
-    "contain demo elements" in {
-      eventually {
-        driver.findElementById("frontend-translations-demo")
-        driver.findElementById("rpc-translations-demo")
-        driver.findElementById("dynamic-rpc-translations-demo")
-      }
-    }
-
     "contain working frontend translations demo" in {
       driver.navigate().refresh()
 
       //runDemo("frontend-translations-demo")()
-      def demo = driver.findElementById("frontend-translations-demo")
+      def demo = findElementById("frontend-translations-demo")
 
       eventually {
         val elements = demo.findElements(new ByCssSelector("li"))
```

Comment:
I guess a `findElement`/`findElements` call on already retrieved Node may throw when that Node's children are not ready yet, however that's almost impossible (so I'd suggest skipping this remark).